pkg/networkmanager/firewall-client.js

```diff
@@ -81,6 +75,23 @@ function initFirewalldDbus() {
         firewall.enabledServices.delete(service);
         firewall.dispatchEvent('changed');
     });
+
+    firewalld_dbus.subscribe({
+        interface: 'org.fedoraproject.FirewallD1',
+        path: '/org/fedoraproject/FirewallD1',
+        member: 'Reloaded'
+    }, (path, iface, signal, args) => {
+        getServices();
+    });
+
+    getDefaultZonePath()
+            .then(path => firewalld_dbus.subscribe({
+                interface: 'org.fedoraproject.FirewallD1.config.zone',
+                path: path,
+                member: 'Updated'
+            }, (path, iface, signal, args) => {
+                firewall.reload();
+            }));
```

Comment:
Please drop the superfluous `{ ; }`. In principle you shouldn't even need a function wrapper at all, just giving `firewall.reload` as the callback should work?